usr/share/rear/output/default/950_copy_result_files.sh

```diff
@@ -46,6 +46,15 @@ cat "$RUNTIME_LOGFILE" > "$TMP_DIR/$final_logfile_name" || Error "Could not copy
 RESULT_FILES+=( "$TMP_DIR/$final_logfile_name" )
 LogPrint "Saving $RUNTIME_LOGFILE as $final_logfile_name to $scheme location"
 
```

Comment:
Could you - by the way - add here a
```
local result_file
```
it is also used in the existing code below at
`if scheme_supports_filesystem $scheme ; then`
but was never marked 'local'.